tests/IntegrationTests/WebAppUiTests/UiTestHelpers.cs

```diff
@@ -9,6 +9,9 @@
 using System.Management;
 using System.Runtime.Versioning;
 using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+using Castle.Core.Internal;

```

Comment:
what are we using Castle.Core.Internal for?

Reply:
Apparently, it was for using IsNullOrEmpty on a dictionary, but I saw I could use identitymodel.tokens instead so I did that.